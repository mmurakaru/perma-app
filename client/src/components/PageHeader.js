import styled from 'styled-components/macro'

export default function PageHeader() {
    return (
        <HeaderWrapper>
            <h1>Overview</h1>
        </HeaderWrapper>
    )
}

const HeaderWrapper = styled.div`
    display: flex;
    justify-content: start;
    width: 100%;
    height: 20px;
    margin-bottom: 20px;
`
