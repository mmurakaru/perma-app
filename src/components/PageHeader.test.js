import PageHeader from './PageHeader'
import { render } from '@testing-library/react'

describe('PageHeader', () => {
    it('renders correctly', () => {
        const { container } = render(<PageHeader />)
        expect(container.firstChild).toMatchSnapshot()
    })
})
