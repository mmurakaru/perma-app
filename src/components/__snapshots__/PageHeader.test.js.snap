// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders correctly 1`] = `
<div
  class="PageHeader__HeaderWrapper-sc-25gwsu-0 iRERYq"
>
  <h1>
    Overview
  </h1>
</div>
`;
